recipes/cytoolz/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.7.5" %}
+
+package:
+  name: cytoolz
+  version: {{ version }}
+
+source:
+  fn: cytoolz-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/cytoolz/cytoolz-{{ version }}.tar.gz
+  md5: 15fbc1c56fca80cbca0e89f37c93d432
+
+build:
+  number: 0
+  script: python setup.py install --record=record.txt
```

Comment:
Oop, that's being changed as well. https://github.com/pytoolz/cytoolz/pull/76/commits/2bfd24227ffd2994dfd781ffe22ddaed4ddeae47. I think the solution might be wait until next cytoolz release, then push this recipe. That should be happening soonish, right @eriknw?